plugins/ReactNativeStyle/ReactNativeStyle.js

```diff
@@ -102,6 +102,10 @@ class NativeStyler extends React.Component {
   _handleStyleRename(oldName: string, newName: string, val: string | number) {
     var style = shallowClone(this.state.style);
     delete style[oldName];
+    if (newName === '') {
+      this.setState({style});
+      return;
```

Comment:
It looks like will not actually work, should keep the `rn-style:rename` event if it rename to `''`, then remove it in RN client.

See [the backend code](https://github.com/facebook/react-devtools/blob/master/plugins/ReactNativeStyle/setupBackend.js#L94).